sw/device/lib/testing/test_framework/ottf_common.ld

```diff
@@ -246,5 +235,18 @@ SECTIONS {
     *(.freertos.heap)
   } > ram_main
 
+   _non_volatile_scratch_size = 2048;
+   _non_volatile_scratch_end = ORIGIN(eflash) + LENGTH(eflash);
+   _non_volatile_scratch_start = _non_volatile_scratch_end - _non_volatile_scratch_size;
+
+  /**
+   * Non-volatile scratch area in flash.
+   */
+  .non_volatile_scratch _non_volatile_scratch_start (NOLOAD) : ALIGN(2048) {
+    KEEP(*(.non_volatile_scratch))
+    KEEP(*(.non_volatile_scratch.*))
+    . = _non_volatile_scratch_size;
+  } > ottf_flash
+
```

Comment:
This commit seems a bit superfluous given the next commit. Maybe squash them.

Reply:
done